mod/wikirate/set/right/citation_count.rb

```diff
@@ -32,6 +39,6 @@ def no_citations? args
   end
 
   def related_overview_card
-    Card.fetch "#{card.cardname.left}+#{Card[:related_articles]}"
+    Card.fetch "#{card.cardname.left}+#{Card[:related_articles].name}"
```

Comment:
I forgot that Name has also a trait method and I didn't want to call `card.left.trait` because I don't know if `left` exists.